### PR TITLE
Translatable option strings

### PR DESCRIPTION
**Pull request by emlys**

### CODE REVIEW

File: src/natcap/invest/crop_production_percentile.py
Comment (phargogh) [inline src/natcap/invest/crop_production_percentile.py:51]: Nice use of a comprehension here!  Is this still a `TODO`, or does the comprehension address it?

File: HISTORY.rst
Comment (phargogh) [inline HISTORY.rst]: These are just with the options presented in the UI, right?  They won't change the `args` option itself, right?  If so, that might be helpful to include here in the HISTORY.

File: src/natcap/invest/recreation/recmodel_client.py
Comment (phargogh) [inline src/natcap/invest/recreation/recmodel_client.py:93]: I think I remember you mentioning that some models (and this looks to be an example) use underscores to separate words ... should these also have a display name set?

**Review by phargogh — CHANGES_REQUESTED** [review]
Hey @emlys I saw [your comment over in the InVEST PR](https://github.com/natcap/invest/pull/645#issuecomment-987210601) and thought I'd take a look, so I'm sorry if I jumped the gun on this!  Just had a couple small comments but overall the changes look good to me.

File: src/natcap/invest/recreation/recmodel_client.py
Comment (emlys) [reply src/natcap/invest/recreation/recmodel_client.py:93]: These should not, because they're column headers in a table, not dropdown options

File: src/natcap/invest/crop_production_percentile.py
Comment (emlys) [reply src/natcap/invest/crop_production_percentile.py:51]: This is still a TODO, since some of the names have abbreviations or suffixes that need to be defined (natcap/invest#614).

File: HISTORY.rst
Comment (emlys) [reply HISTORY.rst]: I did change the actual `args` options here to take advantage of having a separate option key vs. display name. For example, the UI dropdown will show "West Coast of North America and Hawaii" as before, but the arg value will be `"westcoast"`, just so we're not using those long phrases as keys in the model code.

I tried to clarify this in the history note. Let me know what you think!

**Comment by emlys** [comment]
Thanks @phargogh! Yes, this was ready for review. For some reason I couldn't request your review on it, maybe some setting on my fork? Anyways this is ready again.

File: src/natcap/invest/recreation/recmodel_client.py
Comment (phargogh) [reply src/natcap/invest/recreation/recmodel_client.py:93]: Ah right.  Thanks!

File: HISTORY.rst
Comment (phargogh) [reply HISTORY.rst]: The updated history notes look great!  I really like how it clarifies who should take note of the change.

**Review by phargogh — APPROVED** [review]
Ok great!  Thanks for the updates.  I see that there are a couple of build failures @emlys where I think maybe the `investspec` portion of the UG aren't quite lining up with the updated `ARGS_SPEC` for Wave Energy, but that can be resolved back in https://github.com/natcap/invest/pull/645 so I'll merge this one!

**Comment by phargogh** [comment]
Ha, of course I can't merge the branch into your branch on your fork!  So I guess I'll have to leave that to you to do @emlys !

But also I wanted to call out the arg change in the UG, which'll need to be updated over there at some point

**Comment by emlys** [comment]
Oh, right! I just updated the user's guide.

**Pull request merged**
